logstash-rs/src/event.rs

```diff
@@ -0,0 +1,77 @@
+use chrono::{DateTime, Utc};
+use log::Level;
+use serde::Serialize;
+use serde_json::Value;
+use std::{collections::HashMap, time::SystemTime};
+
+#[derive(Debug, Default, Clone, Serialize)]
+pub struct LogStashRecord {
+    #[serde(rename = "@timestamp")]
+    #[serde(with = "logstash_date_format")]
+    timestamp: Option<DateTime<Utc>>,
+    module: Option<String>,
+    file: Option<String>,
+    line: Option<u32>,
+    level: Option<Level>,
+    target: String,
+    #[serde(flatten)]
+    fields: HashMap<String, Value>,
+}
+
+impl LogStashRecord {
+    pub fn new() -> Self {
+        Default::default()
+    }
+
+    pub fn new_with_time_now() -> Self {
```

Comment:
Maybe replace `new()` with this fn. Since current `new()` is the same as `default()`